tests/integration/conftest.py

```diff
@@ -296,7 +296,7 @@ async def app_openstack_runner(
             wait_idle=False,
             use_local_lxd=False,
         )
-    await model.wait_for_idle(apps=[application.name], status=ACTIVE, timeout=90 * 60)
+        await model.wait_for_idle(apps=[application.name], status=ACTIVE, timeout=90 * 60)
```

Comment:
Would this only wait in the else case? I'm wondering if it would be safer to have it wait for both cases to make sure that the existing app is also in an Active status.